### DISCUSSION TITLE
解决 IntelliJ IDEA 启动报 Unsupported Java Version 的问题 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#233] [Announcements]
https://mazhuang.org/2021/03/27/intellij-idea-unsupported-java-version/ 

 今天用我老掉牙的 MacBook Pro 2015 跑 IDEA 时，感觉略卡，于是在网上找了一些教程来优化它的启动和运行速度。

**Comment by zhaochanghui** [comment]
写的不错

**Comment by zhaochanghui** [comment]
测试
